streams/README.md

```diff
@@ -11,6 +11,7 @@ Arrays.stream(persons)
    .average()
    .getAsDouble();
 ```
+## Preparation
 
 1. Run this stream with any `java` you have on your system:
```

Comment:
Why did you make it step 1?  It is just additional information, move it to introduction:

You can run this stream with any `java` you have on your system:
  ```bash
  javac Streams.java
  java Streams 100000 200
  ...
  Iteration 20 finished in 219 milliseconds with checksum e6e0b70aee921601
  TOTAL time: 4717
  ```